islands/formsNossasLojas.tsx

```diff
@@ -0,0 +1,158 @@
+import Icon from "site/components/ui/Icon.tsx";
+
+import { useState } from "preact/compat";
+import { zoom } from "site/islands/Map.tsx";
+import { ItemWithWhatsapp } from "site/sections/NossasLojas/types.ts";
+
+interface StoreInfo {
+  name: string;
+  id: string;
+  address: {
+    postalCode: string;
+    city: string;
+    state: string;
+    neighborhood: string;
+    street: string;
+    number: string;
+    location: {
+      latitude: number;
+      longitude: number;
+    };
+  };
+  businessHours: Array<{
+    dayOfWeek: string;
+    openingTime: string;
+    closingTime: string;
+  }>;
+}
+
+interface Stores {
+  stores: ItemWithWhatsapp[];
+}
+
+const days = [
+    "Domingo",
+    "Segunda-feira",
+    "Terça-feira",
+    "Quarta-feira",
+    "Quinta-feira",
+    "Sexta-feira",
+    "Sábado"
+]
+
+export default function FormsNossasLojas({ stores }: Stores) {
+  const [showFiltered, setShowFiltered] = useState(false);
+  const [queryInput, setQueryInput] = useState("");
+  const [filtered, setFiltered] = useState<ItemWithWhatsapp[]>([]);
+  const [IsCicked, setIsClicked] = useState<number | null>(null);
```

Comment:
1. Mantenha a semântica das variáveis. Nomes iniciados com is... geralmente se tratam de booleanos (isClicked = está clicado?). 
2. Use camelCase para as variáveis.

```suggestion
  const [selectedIndex, setSelectedIndex] = useState<number | null>(null);
```